main/inputoutput/io.py

```diff
@@ -1,2 +1,9 @@
+from zipfile import ZipFile
 def inc(num):
-    return num+2
\ No newline at end of file
+    return num+2
+
+def unzip_file(fromPath,toPath):
+    file=ZipFile(fromPath,'r')
+    file.extractall(toPath)
+    file.close()
+    return 1
```

Comment:
This always returns 1. 

What happens when the file was not inflated?

Steps to reproduce. 

1. Delete the files under /test/data/target
2. run the tests - Tests will pass even though the files are not extracted

<img width="1621" alt="image" src="https://user-images.githubusercontent.com/3809547/54459947-0baf9680-4760-11e9-84c7-5791da69da08.png">